src/views/js/Application.js

```diff
@@ -1,21 +1,16 @@
-import VueMarkdown from 'vue-markdown'
+import EcApplication from './../../components/EcApplication.vue'
 import EcomApps from '@ecomplus/apps-manager'
 
 export default {
   name: 'Application',
   components: {
-    VueMarkdown
+    EcApplication
   },
   data () {
     return {
       application: {},
-      adminSettings: {},
-      applicationObjId: null,
-      showSettings: false,
-      loading: true,
-      installed: false,
-      data: null,
-      hidden_data: null
+      loading: false,
+      localApplication: {}
```

Comment:
`application` e `localApplication` no data da view, por quê?

Reply:
eu tinha imaginado o application como o aplicativo que é feito fetch do market, e o localApplication como o application que vem da store api.

Reply:
Não sei se é necessário porque não sei de cor os objetos das duas APIs, mas se for o caso você deveria nomear isto melhor, algo como `storeApplication` e `marketApplication`...